src/app/components/Results/ResultsList.jsx

```diff
@@ -105,6 +116,10 @@ class ResultsList extends React.Component {
           <span className="nypl-results-room">{location}</span>
           <span className="nypl-results-use">{usageType}</span>
         </div>
+        {
+          !!(items.length === 1) &&
```

Comment:
Do you need the `!!` here? I added it for the other logic checks because if it was false a `0` would get rendered. You should try checking to see. Maybe one logic comparison doesn't warrant a `!!`.